sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuMetrics.scala

```diff
@@ -106,8 +106,6 @@ object GpuMetric extends Logging {
   val FILECACHE_DATA_RANGE_READ_TIME = "filecacheDataRangeReadTime"
   val DELETION_VECTOR_SCATTER_TIME = "deletionVectorScatterTime"
   val DELETION_VECTOR_SIZE = "deletionVectorSize"
-  val CONCAT_HEADER_TIME = "concatHeaderTime"
-  val CONCAT_BUFFER_TIME = "concatBufferTime"
```

Comment:
I checked one of recent runs. Sounds like `concatBufferTime` is still notable?
![Screenshot 2025-02-17 at 13 58 34](https://github.com/user-attachments/assets/9750e916-5b68-43aa-92af-b30f9a43fd8e)

Reply:
Did you check our recent job runs? Is there any outstanding case with buffer concat time worthy an optimization?